all_test.go

```diff
@@ -127,7 +127,7 @@ func TestExportedSymbolsHaveDocs(t *testing.T) {
 	t.Skip()
 
 	err := filepath.WalkDir(".", func(path string, d os.DirEntry, err error) error {
-		if err != nil || d.IsDir() || !strings.HasSuffix(path, ".go") || strings.HasSuffix(path, "_test.go") {
+		if err != nil || d.IsDir() || !strings.HasSuffix(path, ".go") || strings.HasSuffix(path, "_test.go") || strings.HasSuffix(path, ".pb.go") {
```

Comment:
I'm happy to break the line here, but would appreciate suggestions on how/where to do so if so.